EIPS/eip-7652.md

```diff
@@ -0,0 +1,209 @@
+---
+eip: 7652
```

Comment:
```suggestion
eip: 7512
```
Assigning next sequential EIP/ERC number.
EIP/ERC numbering changed to sequential from 7500 and is no longer the PR number.

Please also update the filename.

Reply:
Makes sense. Is this somewhere mentioned? I was reading through the readme, the eip template and [EIP-1](https://eips.ethereum.org/EIPS/eip-1) and couldn't find this. Might be helpful for future submissions ;)

Reply:
Ahh I see it is mentioned on [Ethereum magicians](https://ethereum-magicians.org/t/ethereum-improvement-proposal-all-info/12326). Might be useful to add this to the ERC template :)

Reply:
Nvm I just cannot read, it is part of the template. :see_no_evil: